plugins/de.cau.cs.kieler.klighd.lsp/src/de/cau/cs/kieler/klighd/lsp/model/KGraphModel.xtend

```diff
@@ -29,6 +35,13 @@ import org.eclipse.xtend.lib.annotations.Accessors
 @Accessors
 class SKNode extends SNode {
     List<KGraphData> data
+    HashMap<String, Object> properties = newHashMap
+    int layerId
+    int posId
+    int layerCons
+    int posCons 
+    boolean interactiveLayout
+    Direction direction
```

Comment:
This seems to be very specific to the layered algorithm, should be externalized to some generic property or something similar.

Reply:
You are right, these should be saved in the properties instead (as it is already done).